Assets/__Scripts/MapEditor/AutoSaveController.cs

```diff
@@ -90,6 +92,19 @@ public void OnSave(InputAction.CallbackContext context)
         if (context.performed) CheckAndSave();
     }
 
+    public void OnSaveQuest(InputAction.CallbackContext context)
+    {
+        if (context.performed) CheckAndSaveQuest();
+    }
+
+    public void CheckAndSaveQuest(int _) => CheckAndSaveQuest(); // So it shows up in Unity
+
+    public async void CheckAndSaveQuest(SaveType saveType = SaveType.None)
+    {
+        CheckAndSave();
+        await Exporter.ExportToQuest();
```

Comment:
Note that saving is done on a separate thread. The concern is could this attempt to export while file is still being written to?